### PR TITLE
Add MessageChannel in order to manually send data to App

### PR DESCRIPTION
**Pull request by luisarmandodv**

### CODE REVIEW

**Comment by bitlather** [comment]
@luisarmandodv can you explain why you built out MessageChannel?

What is the relationship with this new dev and IntegrationCable?

**Comment by luisarmandodv** [comment]
@bitlather sure!, basically, I was not able to establish a connection from the App. My main concern was that I was wrongly implementing the setup on my side, so I checked this simple example -> https://github.com/cpunion/TestRNActionCable/tree/master/server

There, it uses `MessageChannel`, I run the code, and it worked, I was able to establish a connection there to that channel. I copied and pasted all that logic in your code Dave, and it is working.

In conclusion, (with my very limited knowledge of RoR) I think it is needed to do some similar implementation of `MessageChannel` for all the other channels you did Dave.

**Comment by bitlather** [comment]
@luisarmandodv interesting; I'll have to look closer. It's possible that the getschwifty gem we use for cables in the webapp doesn't support something like what we're trying to accomplish with the mobile app. Our client code is probably doing some "magic" that I didn't know about, so I was unable to tell you about it. I'll dig a bit more.

**Comment by luisarmandodv** [comment]
Yup, everyone say RoR is like magic haha

Not sure if this would be useful, I'm using this https://www.npmjs.com/package/@rails/actioncable dependency. More exactly this bunch of code -> `https://github.com/rails/rails/blob/master/actioncable/app/assets/javascripts/action_cable.js` in order to connect to the ActionCable. 

This is how I connect to `MessageChannel`
```
const consumer = new Consumer('ws://localhost:3000/cable');

    consumer.subscriptions.create(
      {channel: 'MessageChannel'},
      {
        connected(data) {
        },
        disconnected(data) {
        },
        received(data) {
          debugger
        },
      },
    );
```

**Comment by bitlather** [comment]
@luisarmandodv cables can be used such that multiple users can get the same updates, or every user gets private updates. 

I think the way it's written in this PR, all users will get the same update.

Can you verify the following scenario? (if not, I will try)

- Cable emits a message every 10 seconds with a random 5 character string.
- Browser 1 (eg Chrome) visits the page that shows cable message.
- Browser 2 (eg Chrome Incognito) visits the same page that shows cable message.
- Does Browser 1 and 2 see the same random strings, or do they get different random strings?

**Comment by luisarmandodv** [comment]
I'm afraid I would not be able to do those test, with my knowledge of ruby, that would take me too much time to update the code to do that.

But in my understanding, the response is Yes, as soon as it gets connected, it will perform an object, and it would be the same object every time, as it is hardcoded. 

My recommendation @bitlather is to stop working on this demo harcoded actionCable and focus on the real Epic! integration, because of the deadline.

**Comment by bitlather** [comment]
@luisarmandodv because this is such a deviation from how we developed cables in the past, and this code sample is much smaller and easier to work with, it is more important to have these questions answered now rather than rush the solution.

By adding a random string generator, that would tell us whether or not the same results are being emitted to all subscribers.

I will try to run through this scenario when I'm done running tests against the postgresql upgrade.

**Comment by bitlather** [comment]
I've backed up my original branch as `syncs-timer-getschwifty-integration---dave-bkup`. Merging this PR so I can build out hardcoded cables using a method that magma can interface with via react native.
